### PR TITLE
Suppressing .env loading message when PIPENV_QUIET is set

### PR DESCRIPTION
**Pull request by thunderk**
This is a proposed fix for issue #2358 that have been closed without solution.

The fix suppresses the message "Loading .env environment variables" that is sent to stderr, when PIPENV_QUIET environment variable is set.

This is very useful for people that use _pipenv run_ in cron commands, because writing to stderr may systematically report the cron as error.

### CODE REVIEW

File: tests/unit/test_core.py [hunk 1]
```diff
@@ -57,3 +57,26 @@ def test_load_dot_env_warns_if_file_doesnt_exist(capsys):
...
+def test_load_dot_env_quiet(capsys):
+    # Setting PIPENV_QUIET in environment will suppress the .env loading message
+    message = 'Loading .env environment variables'
+
+    with temp_environ(), TemporaryDirectory(prefix='pipenv-', suffix='') as tempdir:
+        dotenv_path = os.path.join(tempdir.name, 'test.env')
+        with open(dotenv_path, 'w') as f:
+            pass
+
+        with mock.patch('pipenv.environments.PIPENV_DOTENV_LOCATION', dotenv_path):
```
[the remark this answers is not in the document]
Comment (thunderk) [reply tests/unit/test_core.py]: Well I wanted to to this, but the PIPENV_QUIET variable is removed at import-time : https://github.com/pypa/pipenv/blob/1f36ca17bc0511492d697028770abf9daf600a97/pipenv/environments.py#L271

Do you think patching PIPENV_VERBOSITY variable is acceptable instead ?

[the remark this answers is not in the document]
Comment (thunderk) [reply tests/unit/test_core.py]: Sorry I meant this line : https://github.com/pypa/pipenv/blob/1f36ca17bc0511492d697028770abf9daf600a97/pipenv/environments.py#L270

[the remark this answers is not in the document]
Comment (frostming) [reply tests/unit/test_core.py]: Okay, setting `PIPENV_VERBOSITY` to `-1` looks fine to me.

[the remark this answers is not in the document]
Comment (thunderk) [reply tests/unit/test_core.py]: Ok thanks, changes done

[the remark this answers is not in the document]
Comment (thunderk) [reply tests/unit/test_core.py]: I can squash in one commit if you prefer

[the remark this answers is not in the document]
Comment (frostming) [reply tests/unit/test_core.py]: I don't have preference on this. Either is ok for me

[the remark this answers is not in the document]
Comment (thunderk) [reply tests/unit/test_core.py]: Ok, I just pushed the squashed commit

**Comment by jxltom** [comment]
Why the CI in windows for python27 can't pass...

**Comment by thunderk** [comment]
I don't know why CI fails on windows, #3426 has the same error.

It fails while creating the virtualenv, with the error:
UnicodeDecodeError: 'utf8' codec can't decode byte 0xeb in position 5: invalid continuation byte

Log is here : https://dev.azure.com/pypa/pipenv/_build/results?buildId=3852&view=logs&jobId=b63f28c9-dd61-5e61-2539-dacb59be920f&taskId=838050e5-b1f9-5e09-c266-e53e847c68d9&lineStart=75&lineEnd=76&colStart=1&colEnd=1

I'm not sure how this could be related with my changes, they do not include non-ascii characters.

**Comment by jxltom** [comment]
Your PR looks good to me. I don't think the failing CI is related with your PR since it is a so minor change. But it still worth investigating why the CI fails...

**Comment by jxltom** [comment]
One more thing to say is that we might still need to make the test pass first, otherwise merging this may make our CI failing all the time.

**Comment by thunderk** [comment]
@jxltom Sure, I added the news, I hope it's ok

**Comment by thunderk** [comment]
@techalchemy Yes, I use an editor aware of .editorconfig. I checked again, and the files I edited seems to be saved as utf-8. I will try to add the coding:utf-8 header in the test file (it was missing, but I can't see non utf-8 chars in my diff).

**Comment by thunderk** [comment]
OK, I'm stuck, I tried to go back from master and rewrite my changes with another editor (just to be sure). Also checked the modified files with some utf-8 validator, and still no luck...
I don't have a Windows installation at hand to reproduce.

**Comment by techalchemy** [comment]
well, note that all tests were failing until recently, so that may be contributing... the changes seem ok to me so i'll just update your pr with the current changes and see if that works

**Comment by techalchemy** [comment]
(I think the transient test failure issue is resolved and this is probably something we can merge now!)

**Comment by ExaneServerTeam** [comment]
Any news on this item. I am also interested to get ride this notification.

**Comment by matteius** [comment]
This was fixed in these recent PR:  
https://github.com/pypa/pipenv/pull/5006
https://github.com/pypa/pipenv/pull/5010

**Pull request closed without merging**